launch/app/app.bndrun

```diff
@@ -296,4 +248,52 @@ feature.openhab-model-runtime-all: \
 	org.openhab.core.voice;version='[4.2.0,4.2.1)',\
 	org.openhab.ui;version='[4.2.0,4.2.1)',\
 	org.openhab.ui.basic;version='[4.2.0,4.2.1)',\
-	org.openhab.ui.iconset.classic;version='[4.2.0,4.2.1)'
+	org.openhab.ui.iconset.classic;version='[4.2.0,4.2.1)',\
+	com.google.guava;version='[33.0.0,33.0.1)',\
+	com.google.guava.failureaccess;version='[1.0.2,1.0.3)',\
+	com.sun.jna;version='[5.14.0,5.14.1)',\
```

Comment:
I always keep the bundles sorted as sometimes the bundle order causes very slow bundle resolution, see: https://community.openhab.org/t/starting-demo-app-takes-ages-10-12min-on-every-run-osgi-startup/122083